### ISSUE TITLE
Score only and a blank white page

### ISSUE THREAD

**Issue by j3douglas** [kubowania/memory-game#23]
Hey, I've gone over my code a lot and I can't seem to find out why I keep getting a white page with just the word "score" at the top. I've seen other people asking this question online but so far I haven't gotten an answer. HELP!

**Comment by j3douglas** [comment]
I figured it out! I named my image for the back of the cards "cover.png" instead of "blank.png' and never renamed it in the card.setAttribute method on the style.css page. Hopes this helps anyone who may have had the same question. Your problem could be different but I guess my point is to not give up and keep working through it. You'll find the problem! In this case I just had to walk away from it for a bit.

**Comment by j3douglas** [comment]
I figured it out! I named my image for the back of the cards "cover.png" instead of "blank.png' and never renamed it in the card.setAttribute method on the style.css page. Hopes this helps anyone who may have had the same question. Your problem could be different but I guess my point is to not give up and keep working through it. You'll find the problem! In this case I just had to walk away from it for a bit.
